src/__tests__/SelectedRevisionsTable.test.tsx

```diff
@@ -1,286 +0,0 @@
-import { renderHook } from '@testing-library/react';
```

Comment:
I see a lot of files like this one are deleted, maybe you meant to just skip the tests instead of delete them? In that case you can use `describe.skip` instead of just `describe`, and keep the file in this patch.

Of course we'll want to unskip it later, but I'm fine if you want to work on it in a dedicated follow-up PR.

(Same comment for other deleted test files)

Reply:
Honestly, these tests and reducers need be to redone. We will have to create new tests when these beta versions of these components are made. And like you said before, the history is there on github.

Reply:
My next PR will focus on SelectedRevisions so all these tests will pop back again.